ecl/hthor/hthorkey.cpp

```diff
@@ -974,8 +974,9 @@ const void *CHThorIndexReadActivity::nextRow()
 }
 
 
-const void *CHThorIndexReadActivity::nextGE(const void * seek, unsigned numFields)
+const void *CHThorIndexReadActivity::nextRowGE(const void * seek, unsigned numFields, bool &wasCompleteMatch, const SmartStepExtra &stepExtra)
 {
+    // MORE - should set wasCompleteMatch
```

Comment:
Actually, it looks like all calls set it to true before calling, so safe as it stands.  Should delete or modify the comment in a subsequent commit.